cmd/enola/find.go

```diff
@@ -43,3 +54,62 @@ func findAndShowResult(username, site string) {
 		os.Exit(1)
 	}
 }
+
+func writeOutput(m *model, outputPath string, filename string, printFound bool) error {
```

Comment:
Just a nitpicking, But I would prefer to have these file related functions in a separate file.